example/bayesian-methods/data_loader.py

```diff
@@ -29,8 +29,7 @@ def load_mnist(training_num=50000):
             'https://github.com/sxjscience/mxnet/raw/master/example/bayesian-methods/mnist.npz'
         )
         print('Downloading data from %s to %s' % (origin, data_path))
-        context = ssl._create_unverified_context()
-        urllib.request.urlretrieve(origin, data_path, context=context)
+        urllib.request.urlretrieve(origin, data_path)
```

Comment:
I meet an error such as "Unverified SSL connection" previously when directly using urlretrieve.

Reply:
But it seems that there's no `context` arg for `urllib.request.urlretrieve`. I change to a new approach.